### PR TITLE
IDEX-1605: add new feature 'Got ot implementation'

### PR DESCRIPTION
**Pull request by svor**

### CODE REVIEW

File: plugin-java/che-plugin-java-ext-lang-client/src/main/java/org/eclipse/che/ide/ext/java/client/navigation/openimplementation/OpenImplementation.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+import java.util.List;
+
+/**
+ * The visual part of window which contains implementing nodes.
+ *
+ * @author Valeriy Svydenko
+ */
+@ImplementedBy(OpenImplementationImpl.class)
+interface OpenImplementation extends View<OpenImplementation.ActionDelegate> {
```
Comment (azatsarynnyy) [inline plugin-java/che-plugin-java-ext-lang-client/src/main/java/org/eclipse/che/ide/ext/java/client/navigation/openimplementation/OpenImplementation.java]: I think that name ```OpenImplementationView``` will be better for this class

File: plugin-java/che-plugin-java-ext-lang-client/src/main/java/org/eclipse/che/ide/ext/java/client/navigation/openimplementation/OpenImplementationImpl.java [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+
+import static org.eclipse.che.ide.ui.smartTree.TreeSelectionModel.Mode.SINGLE;
+
+/**
+ * Implementation of {@link OpenImplementation} view.
+ *
+ * @author Valeriy Svydenko
+ */
+@Singleton
+final class OpenImplementationImpl extends Window implements OpenImplementation {
```
Comment (azatsarynnyy) [inline plugin-java/che-plugin-java-ext-lang-client/src/main/java/org/eclipse/che/ide/ext/java/client/navigation/openimplementation/OpenImplementationImpl.java]: I think that name*OpenImplementationViewImpl* will be better for this class

File: plugin-java/che-plugin-java-ext-lang-client/src/main/java/org/eclipse/che/ide/ext/java/client/navigation/openimplementation/OpenImplementation.java (hunk 1 above)
Comment (vparfonov) [inline plugin-java/che-plugin-java-ext-lang-client/src/main/java/org/eclipse/che/ide/ext/java/client/navigation/openimplementation/OpenImplementation.java]: +1

File: plugin-java/che-plugin-java-ext-lang-client/src/main/java/org/eclipse/che/ide/ext/java/client/navigation/openimplementation/OpenImplementationImpl.java (hunk 2 above)
Comment (vparfonov) [inline plugin-java/che-plugin-java-ext-lang-client/src/main/java/org/eclipse/che/ide/ext/java/client/navigation/openimplementation/OpenImplementationImpl.java]: +1

**Comment by vparfonov** [comment]
ok

**Pull request merged**